tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_backward_ops.cpp

```diff
@@ -1783,6 +1784,31 @@ namespace tt::tt_metal::detail{
                 "all_dimensions", "Consider all dimension (ignores ``dim`` param)", "bool", "", "Yes"
                 "dim", "Dimension to perform prod", "int", "", "Yes"
                 "output_mem_config", "Layout of tensor in TT Accelerator device memory banks", "MemoryConfig", "Default is interleaved in DRAM", "No"
+                "are_required_outputs", "Boolean values for the required outputs: input_grad", "List of bool", "Default value is [True]", "No"
+                "input_grad", "optional output tensor", "Tensor", "default is None", "No"
+        )doc");
+
+    m_tensor.def("prod_bw", py::overload_cast<uint8_t, const Tensor&, const Tensor&, bool, int64_t, const MemoryConfig&, const std::vector<bool>&, std::optional<Tensor> >(&tt::tt_metal::prod_bw),
+            py::arg("cq_id") = 0, py::arg("grad").noconvert(), py::arg("input").noconvert(), py::arg("all_dimensions") , py::arg("dim") , py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, py::arg("are_required_outputs").noconvert() = std::vector<bool>{true}, py::arg("input_grad").noconvert() = std::nullopt, R"doc(
```

Comment:
Please change `cq_id` from a positional to a kw_only argument called `queue_id`.

CC @eyonland